plugin.json

```diff
@@ -1123,13 +1123,15 @@
       "required": true
     },
     {
-      "name": "csvFile",
-      "description": "csvfile : singlecsv for saving values in one csv file and separate csv to save values for each image in separate csv file",
+      "name": "outputType",
+      "description": "outputType : singlecsv for saving values in one csv file and separate csv to save values for each image in separate csv file, arrow and parquet for saving in those file formats respectively",
```

Comment:
How about "..., arrow and parquet for saving in Apache Arrow and Parquet file formats respectively" since users are massively unaware of Apache formats ?

Reply:
updated as requested